tests/CommandsTest.php

```diff
@@ -117,41 +115,21 @@
     expect(Schema::hasTable('users'))->toBeFalse();
 });
 
-test('seed command works', function () {
-    $this->markTestIncomplete();
-});
+// Incomplete test
+test('seed command works');
 
 test('database connection is switched to default', function () {
-    $originalDBName = DB::connection()->getDatabaseName();
-
-    Artisan::call('tenants:migrate');
-    expect(DB::connection()->getDatabaseName())->toBe($originalDBName);
-
-    Artisan::call('tenants:seed', ['--class' => ExampleSeeder::class]);
-    expect(DB::connection()->getDatabaseName())->toBe($originalDBName);
-
-    Artisan::call('tenants:rollback');
-    expect(DB::connection()->getDatabaseName())->toBe($originalDBName);
-
-    $this->run_commands_works();
-    expect(DB::connection()->getDatabaseName())->toBe($originalDBName);
+    databaseConnectionSwitchedToDefault();
 });
 
 test('database connection is switched to default when tenancy has been initialized', function () {
     tenancy()->initialize(Tenant::create());
 
-    $this->database_connection_is_switched_to_default();
+    databaseConnectionSwitchedToDefault();
 });
 
 test('run commands works', function () {
```

Comment:
Unrelated to your changes, but seems like this test name has a typo. Should be:
```suggestion
test('run command works', function () {
```